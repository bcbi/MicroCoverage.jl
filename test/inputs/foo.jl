function foo(x)
    if x == 1 || x == 100
        return "hello"
    else
        return "goodbye"
    end
end
