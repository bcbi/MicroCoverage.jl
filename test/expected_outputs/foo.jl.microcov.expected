[1,1,1]                 function foo(x)
[1,0,1,0,1]                 if x == 1 || x == 100
[1]                             return "hello"
-                           else
[0]                             return "goodbye"
-                           end
-                       end
-                       
